recipes/oeel/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "oeel" %}
+{% set version = "0.9.7.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oeel-{{ version }}.tar.gz
+  sha256: aa17434e8ef0fd2eccc4e22554b7cbda33e7473f7fdc8275a0c158757f09d77e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - earthengine-api
+    - pyzmq
+    - git
+    - nodejs >=16
```

Comment:
If you require nodejs at run is there nothing that needs to be compiled to JavaScript before?